src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -400,9 +402,9 @@ export default defineComponent({
 
       multisigTx.value = {
         ...activeMultisigTx.tx,
-        type: SCHEMA.TX_TYPE.gaMeta,
-        tag: SCHEMA.TX_TYPE.gaMeta,
-      } as ITx;
+        type: Tag[Tag.GaMetaTx],
+        tag: Tag.GaMetaTx,
+      } as any;
```

Comment:
Maybe there is a way for not using the `as any` here?
For example the `tag` property of the `ITx` requires `string`, but the `Tag.GaMetaTx` is a number. Is this a mistake?

Reply:
The problem here is that ITx is perfectly describing the middleware transactions, but if we are going to use the type that is returned from `unpackTx` like in this particular case, we would need a deep refactoring of the whole project. And i suggest to split this into different task.

Reply:
OK, so please add `TODO` comment that describes why and what needs to be refactored and then resolve my comment.